extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -83,8 +93,12 @@ export class BindExpression {
     /** @const {string} */
     this.expressionString = expressionString;
 
-    /** @const {!./bind-expr-defines.AstNode} */
+    /** @const @private {!./bind-expr-defines.AstNode} */
     this.ast_ = parser.parse(this.expressionString);
+
+    /** @const @private {!Object} */
+    this.builtInScope_ = Object.create(null);
+    this.builtInScope_['Math'] = Math;
```

Comment:
Done. We might want to add static functions from other built-in objects in the future, e.g. `Date.now()`, but I'm not aware of any overlapping function names.